src/client/unified-form/common/entity-modal-body.tsx

```diff
@@ -0,0 +1,89 @@
+import {EntityModalBodyProps, EntityModalDispatchProps} from '../interface/type';
+import AliasModalBody from '../../entity-editor/alias-editor/alias-modal-body';
+// import {validateAliases, validateIdentifiers, validateNameSection} from '../../entity-editor/validators/common';
+import AnnotationSection from '../../entity-editor/annotation-section/annotation-section';
+import IdentifierModalBody from '../../entity-editor/identifier-editor/identifier-modal-body';
+import NameSection from '../../entity-editor/name-section/name-section';
+import React from 'react';
+import RelationshipSection from '../../entity-editor/relationship-editor/relationship-section';
+import SingleAccordion from './single-accordion';
+import SubmissionSection from '../../entity-editor/submission-section/submission-section';
+import {connect} from 'react-redux';
+import {omit} from 'lodash';
+import {removeEmptyAliases} from '../../entity-editor/alias-editor/actions';
+import {removeEmptyIdentifiers} from '../../entity-editor/identifier-editor/actions';
+// import {validateAuthorSection} from '../../entity-editor/validators/author';
```

Comment:
What was your final outcome with the validation state? Was it still slowing down the app too much? If so we'll unfortunately have to get rid fo the commented out code (and any other code that was just for this validation feature) [a moment of silence for the fallen bits].

Reply:
Though after optimization this doesn't seem to cause any issue. but now i am not sure if we need validations on modal, let me know if you think we should keep the validations or not.